app/settings/settings.json

```
@@ -1,5 +1,5 @@
 {
-    "current_theme": "Dark",
+    "current_theme": "Larissa",
```

Mantenha o tema `Dark` como padrão:

```suggestion
    "current_theme": "Dark",
```